src/ui/util.coffee

```diff
@@ -191,8 +192,19 @@ insertTextAtCursor = (el, text) ->
         range.text = text
         range.select()
 
+# AutoLaunch requires a path unless you are running in electron/nw
+vesrions = process?.versions
+if versions? and (versions.nw? or versions['node-webkit']? or versions.electron?)
+    autoLaunchPath = undefined
+else
+    autoLaunchPath = process.cwd()
```

Comment:
why not use `process.execPath()` as the default in auto-launch?

![image](https://cloud.githubusercontent.com/assets/211358/24049150/0a3bddde-0b23-11e7-8709-0571bbf41e98.png)